api/pkg/s3/listbuckets.go

```diff
@@ -41,6 +41,7 @@ func parseListBuckets(list *s3.ListBucketsResponse) []byte {
 	for _, value := range list.Buckets {
 		ctime := time.Unix(value.CreateTime, 0).Format(time.RFC3339)
 		versionOpts := model.VersioningConfiguration{}
+		versionOpts.Status = utils.VersioningDisabled
```

Comment:
Hi, why always 'disabled' here?

Reply:
default is disabled. then it gets set to whatever the option on the bucket is, if any